docs/source/configuration/overview.mdx

```diff
@@ -83,7 +81,7 @@ This reference lists and describes the options supported by the `router` binary.
 
 ##### `-s` / `--supergraph`
 
-`APOLLO_ROUTER_SUPERGRAPH_PATH`
+`APOLLO_ROUTER_SUPERGRAPH_PATH`, `APOLLO_ROUTER_SUPERGRAPH_URLS`
```

Comment:
Let's also explain the difference between these and indicate that URLs are comma separated.
These URLS will potentially end up in log messages, so users should take care not to embed tokens in the URL.